src/ServiceManager/Travely.ServiceManager.DAL/ServiceManagerDbContext.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Travely.ServiceManager.Abstraction.Models.Db;
+
+namespace Travely.ServiceManager.DAL.Data
```

Comment:
".Data" can be removed

Reply:
The same for src/ServiceManager/Travely.ServiceManager.DAL/ServiceManagerDbContextFactory.cs